components/basic-story-hero/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "basic-story-hero",
-  "version": "0.0.10",
```

Comment:
Is 0.0.11 already in `develop` @iamrentman ? Just curious about the 10 to 12 increment.